sonar-php-plugin/src/main/resources/org/sonar/plugins/php/reports/phpstan/rules.json

```diff
@@ -1,6 +1,12 @@
 [
   {
     "key": "phpstan.finding",
-    "name": "PHPStan Rule"
+    "name": "PHPStan Rule",
```

Comment:
Can only be tested when https://github.com/SonarSource/sonar-php/pull/1268 is merged, depending what is reviewed before, I will add the test to the other branch.